### PR TITLE
cd: add Italian translation

### PR DESCRIPTION
**Pull request by MrTriad**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.it/windows/cd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Mostra informazioni riguardo alla directory corrente e permette di muoversi in un altra directory.
+> Ulteriori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/cd>.
```
Comment (kbdharun) [inline pages.it/windows/cd.md:4]:
```suggestion
> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/cd>.
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

File: pages.it/windows/cd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`cd`
+
+- Spostati nella directory superiore:
+
+`cd ..`
+
+- Spostati ad una specifica directory sullo stesso disco:
+
+`cd {{percorso\alla\directory}}`
```
Comment (kbdharun) [inline pages.it/windows/cd.md:16]: I am not sure about the path. I will request a review from a native speaker. (Ref: 9031)

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.it/windows/cd.md (hunk 2 above)
Comment (MrTriad) [reply pages.it/windows/cd.md:16]: That should be right and easy to test, though it's a fair doubt. Thanks for the suggestions ^^

File: pages.it/windows/cd.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Mostra informazioni riguardo alla directory corrente e permette di muoversi in un altra directory.
+> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/cd>.
+
+- Mostra il percorso(path) della directory corrente:
```
Comment (mebeim) [inline pages.it/windows/cd.md:6]:
```suggestion
- Mostra il percorso (path) della directory corrente:
```

File: pages.it/windows/cd.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`cd`
+
+- Spostati nella directory superiore:
+
+`cd ..`
+
+- Spostati ad una specifica directory sullo stesso disco:
+
+`cd {{percorso\alla\directory}}`
```
Comment (mebeim) [inline pages.it/windows/cd.md:16]:
```suggestion
`cd {{percorso\della\directory}}`
```

For paths in Italian, we are in the process of translating all instances of "al", "alla", "allo", etc. into "del", "della", "dello". We can use "della" here.

File: pages.it/windows/cd.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Spostati ad una specifica directory in un altro disco:
+
+`cd /d {{C}}:{{percorso\alla\directory}}`
```
Comment (mebeim) [inline pages.it/windows/cd.md:20]:
```suggestion
`cd /d {{C}}:{{percorso\della\directory}}`
```
Same here.

**Review by mebeim — CHANGES_REQUESTED** [review]

File: pages.it/windows/cd.md (hunk 4 above)
Comment (kbdharun) [reply pages.it/windows/cd.md:16]: > For paths in Italian, we are in the process of translating all instances of "al", "alla", "allo", etc. into "del", "della", "dello". We can use "della" here.


Thanks @mebeim , I will use this as a reference for future suggestions.

**Comment by MrTriad** [comment]
Resolved the chances requested. Thanks @mebeim for the useful suggestions ^^
I'll try to stick to these guidelines in the next commits.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by mebeim — APPROVED** [review]

**Pull request merged**